derive/src/util.rs

```diff
@@ -59,23 +59,41 @@ pub fn optional_attribute_arg(
     Ok(arg_value)
 }
 
-pub fn module_class_name(mod_name: Option<String>, class_name: &str) -> String {
+pub(crate) fn module_class_name(mod_name: Option<String>, class_name: &str) -> String {
     if let Some(mod_name) = mod_name {
         format!("{}.{}", mod_name, class_name)
     } else {
         class_name.into()
     }
 }
 
```

Comment:
We can just use the `str.strip_prefix()` method, it was stabilized in 1.45

Reply:
Sorry, I didn't get what you want to tell. It looks like optional concatnation rather than stripping.

Reply:
Oh, whoops, meant to comment this on the `util::strip_prefix` function; the app I use isn't great about being precise like that :sweat_smile: